package.json

```diff
@@ -64,9 +64,11 @@
     "moment": "2.17.1",
     "promise": "7.1.1",
     "react": "15.4.1",
+    "react-addons-shallow-compare": "15.4.1",
```

Comment:
Is this explicitly stated because it's required by react-virtualized? Because I can't see it used anywhere except in `webpack.config.dll.js`.
Is there a reason why PureComponent couldn't be used instead?